smrf/data/mysql_data.py

```diff
@@ -95,6 +95,7 @@ def get_data(self, table, station_ids, start_date, end_date, variables):
             start_date: start of time period
             end_date: end of time period
             variable: string for variable to get
+            time_zone: String timezone to set the data in
```

Comment:
The database time zone is UTC. If a user passed MST to SMRF for the `time_zone` this would then be set as MST and be off, correct? Maybe a little more doc's on what this time zone value is and what it does.

Reply:
Yep, that's why I didn't add it to the class init but the get_data function.It seemed more intuitive that way to me.  As for docs, sure. Are you looking for something more like:

```
time_zone: string name of a valid pytz time zone to convert the database data to, default UTC
```

Reply:
I think I'm just confusing myself. The time zone in the core config is for the provided times. A user shouldn't be specifying the data in UTC and run times in MST. They could but that's not the point of the time zone config